ios/FluentUI/SegmentedControl/SegmentedControl.swift

```diff
@@ -251,6 +279,8 @@ open class SegmentedControl: UIControl {
             addSubview(bottomSeparator)
             addSubview(selectionView)
         }
+
+        setupLayoutConstraints()
     }
 
     public required init?(coder aDecoder: NSCoder) {
```

Comment:
line 298, it should just be "let button: UIButton" so we aren't creating empty button then actually create a tab or switch button.